### ISSUE TITLE
Pass around actual (unix-style) paths rather than parent paths

### ISSUE THREAD

**Issue by andylolz** [andylolz/pyandi#9]
This would be way more flexible, as we could construct a DatasetSet of **all** datasets by doing something like this:

```python
pyandi.DatasetSet(data_path='__pyandicache__/registry/data/*/*')
```

At present, there’s no way to do this.

**Comment by andylolz** [comment]
Ooph, this would be really good.

**Comment by andylolz** [comment]
Fixed in 67442f9.
